benchmark/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.library'
```

Comment:
move the benchmark project into a top level directory called "BenchmarkSample"

That'll give us room to do more, separate studio projects.